### DISCUSSION TITLE
Error after migrate from v2 to v3 using migration tool

### DISCUSSION THREAD

**Discussion by Sartoric** [livewire/livewire#6659] [7. Bugs (archived)]
### Livewire version

3.0.1

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Just migrated a project with some LW 2.x component

I started getting this error (in console) while typing in a textarea inlcuded in a LW component
`TypeError: window.Alpine.cloneNode is not a function. (In 'window.Alpine.cloneNode(from, toEl)', 'window.Alpine.cloneNode' is undefined)`

apparently (or I missed that step) the migration tool doesn't check the app.js file for a manual load of Alpine

Removing it seems to fix the issue 

```
// import Alpine from 'alpinejs';
// import focus from '@alpinejs/focus';

// window.Alpine = Alpine;
//
// Alpine.plugin(focus);
//
// Alpine.start();
```

### Additional information

as far as I remember the app.js was built by jetstream while creating the fresh project 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by lsmith77** [comment]
Yeah you need to remove it https://livewire.laravel.com/docs/upgrading#alpinejs
